### ISSUE TITLE
Training our own dataset with circular annotations.

### ISSUE THREAD

**Issue by ahsan44411** [kentaroy47/frcnn-from-scratch-with-keras#49]
I have a cancer dataset with a center pixel circular dataset along with the radius of the circle. Does your model only take data as 'path  x1,y1,x2,y2 class' or can I change the format for my needs?

**Comment by kentaroy47** [comment]
you have to change the localization layers to output the center pixels.

or you can generate circles instead of rectangles using x1,y1,x2,y2 information to save effort

**Comment by ahsan44411** [comment]
What is the current version of tensorflow-gpu does this support? Doesn't seem to work with tensorflow-gpu-1.12.1

**Comment by kentaroy47** [comment]
can you try with 1.12.0? that is the version I used.

would be helpful if you can post error comments.

**Comment by devendraswamy** [comment]
could you please tell about the tuning parameters ? I am  already  train the model with my custom data and I want to increase the model accuracy , so please tell me about model tuning parameters,

sir one more , this training goes very slowly and could you tell me about how can I increase the training speed ??
